include/ajax.tickets.php

```diff
@@ -635,12 +635,16 @@ function editField($tid, $fid) {
                     case $field instanceof DepartmentField:
                         $clean = (string) Dept::lookup($field->getClean());
                         break;
+                    case $field instanceof TextareaField:
+                        $clean =  (string) $field->getClean();
+                        $clean = ($clean && ($clean != '<p><br /></p>')) ? $clean : '&mdash;' . __('Empty') .  '&mdash;';
```

Comment:
Instead, use Format::strip_tags and if null or empty then return EMPTY otherwise use clean.